### PR TITLE
Rename axios wrapper calls

### PR DESCRIPTION
**Pull request by rtorrero**
Pretty much self explanatory :-)

### CODE REVIEW

**Review by dottorblaster — APPROVED** [review]

File: assets/js/components/ClustersList.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { addTagToCluster, removeTagFromCluster } from '@state/clusters';
 import ClusterLink from '@components/ClusterLink';
 import { ExecutionIcon } from '@components/ClusterDetails';
 import { ComponentHealthSummary } from '@components/HealthSummary';
-import { axiosPost, axiosDelete } from '@lib/network';
+import { post, del } from '@lib/network';
```
Comment (nelsonkopliku) [inline assets/js/components/ClustersList.jsx:9]: nitpick: `del`-`ete` :smile: 
Anyway, as you wish.

**Review by nelsonkopliku — APPROVED** [review]

File: assets/js/components/ClustersList.jsx (hunk 1 above)
Comment (dottorblaster) [reply assets/js/components/ClustersList.jsx:9]: Actually I was gonna tell you the same :joy: but really, your call

Comment (rtorrero) [reply assets/js/components/ClustersList.jsx:9]: It's a reserved word :/

Comment (dottorblaster) [reply assets/js/components/ClustersList.jsx:9]: aaah damn you're right :frowning:

Comment (nelsonkopliku) [reply assets/js/components/ClustersList.jsx:9]: oh noes!
Whatever it works.

Comment (dottorblaster) [reply assets/js/components/ClustersList.jsx:9]: Leave it as it is, we can refactor it later if we find a way, no worries!

**Pull request merged**